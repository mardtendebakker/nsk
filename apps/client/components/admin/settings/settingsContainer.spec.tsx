import { render } from '@testing-library/react';
import SettingsContainer from './settingsContainer';

jest.mock('next/router', () => ({
  useRouter: () => ({ pathname: '' }),
}));

describe('settingsContainer', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<SettingsContainer><div /></SettingsContainer>);
    expect(asFragment()).toMatchSnapshot();
  });
  it('renders children', () => {
    const { getByText } = render(<SettingsContainer><div>child</div></SettingsContainer>);
    expect(getByText('child')).not.toBeNull();
  });
});
