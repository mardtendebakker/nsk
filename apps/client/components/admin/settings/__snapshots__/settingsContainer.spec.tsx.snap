// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`settingsContainer matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardContent-root css-1u0k5eo-MuiCardContent-root"
    >
      <div
        class="MuiBox-root css-8joivn"
      >
        <ul
          class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
          role="menu"
          tabindex="-1"
        >
          <a
            href="/admin/settings/locations"
            style="text-decoration: none;"
          >
            <li
              class="MuiButtonBase-root MuiMenuItem-root MuiMenuItem-gutters MuiMenuItem-root MuiMenuItem-gutters css-10xdxf9-MuiButtonBase-root-MuiMenuItem-root"
              role="menuitem"
              tabindex="-1"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
              >
                locations
              </p>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </li>
          </a>
          <a
            href="/admin/settings/product-types"
            style="text-decoration: none;"
          >
            <li
              class="MuiButtonBase-root MuiMenuItem-root MuiMenuItem-gutters MuiMenuItem-root MuiMenuItem-gutters css-10xdxf9-MuiButtonBase-root-MuiMenuItem-root"
              role="menuitem"
              tabindex="-1"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
              >
                productTypes
              </p>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </li>
          </a>
          <a
            href="/admin/settings/attributes"
            style="text-decoration: none;"
          >
            <li
              class="MuiButtonBase-root MuiMenuItem-root MuiMenuItem-gutters MuiMenuItem-root MuiMenuItem-gutters css-10xdxf9-MuiButtonBase-root-MuiMenuItem-root"
              role="menuitem"
              tabindex="-1"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
              >
                attributes
              </p>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </li>
          </a>
          <a
            href="/admin/settings/tasks"
            style="text-decoration: none;"
          >
            <li
              class="MuiButtonBase-root MuiMenuItem-root MuiMenuItem-gutters MuiMenuItem-root MuiMenuItem-gutters css-10xdxf9-MuiButtonBase-root-MuiMenuItem-root"
              role="menuitem"
              tabindex="-1"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
              >
                tasks
              </p>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </li>
          </a>
          <a
            href="/admin/settings/product-statuses"
            style="text-decoration: none;"
          >
            <li
              class="MuiButtonBase-root MuiMenuItem-root MuiMenuItem-gutters MuiMenuItem-root MuiMenuItem-gutters css-10xdxf9-MuiButtonBase-root-MuiMenuItem-root"
              role="menuitem"
              tabindex="-1"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
              >
                productStatuses
              </p>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </li>
          </a>
          <a
            href="/admin/settings/order-statuses"
            style="text-decoration: none;"
          >
            <li
              class="MuiButtonBase-root MuiMenuItem-root MuiMenuItem-gutters MuiMenuItem-root MuiMenuItem-gutters css-10xdxf9-MuiButtonBase-root-MuiMenuItem-root"
              role="menuitem"
              tabindex="-1"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
              >
                orderStatuses
              </p>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </li>
          </a>
        </ul>
      </div>
      <div
        class="MuiBox-root css-1p0to43"
      >
        <div />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
